en/sensor/airspeed.md

```diff
@@ -19,7 +19,9 @@ Recommended digital airspeed sensors include:
 * [Sensirion SDP3X Differential Pressure Sensor](https://www.sensirion.com/en/flow-sensors/differential-pressure-sensors/worlds-smallest-differential-pressure-sensor/)
 * [Holybro Digital Air Speed Sensor](https://shop.holybro.com/digital-air-speed-sensor_p1029.html)
 
-All the sensors are connected via the I2C bus/port.
+All the above sensors are connected via the I2C bus/port.
+
+> **Note** Additionally, the [Avionics Anonymous Air Data Computer](https://www.tindie.com/products/avionicsanonymous/uavcan-air-data-computer/) can be connected to the UAVCAN bus to determine the true static pressure (via air temperature measurement and a barometer).
```

Comment:
I modified this a bit. Please confirm:
1. True static pressure information can be used with the other airspeed sensors to get better airspeed estimate. Is that the point of this?
2. Does PX4 do everything else needed to use this computer when it is attached? 
3. Does this also need a separate temperature measurement sensor and barometer, and are they provided?

Essentially confirm what is needed to make this work and why you bother.

Reply:
Thanks for the improvements!

1 and 3. Not quite - this devices includes its own differential pressure sensor, barometer, and outside air temp probe. The static pressure measurement it provides can't be used to improve airspeed measured by a separate device, but the plumbed static source to the barometer onboard this device gives you a much more accurate static pressure source for high-accuracy altitude, as long as you plumb it to a good static pressure source on the vehicle, i.e. on an air data boom. Does that make sense?
2. Yes, it's plug and play as long as UAVCAN is enabled. PX4 will recognize it as both an airspeed or differential pressure source (can be configured as either) as well as a barometer.

Reply:
Great, the new words look good.